provisioning/service/samples/readme.md

```diff
@@ -0,0 +1,72 @@
+# Samples for the Azure IoT Device Provisioning Service SDK for Node.js
+
+This file can be found in http://github.com/azure/azure-iot-sdk-node/provisioning/service/samples
+
+This folder contains simple samples showing how to use the various features of the Microsoft Azure IoT Device Provisioning Service from an application running JavaScript code.
+
+## List of samples
+
+* Create a simple IndividualEnrollment object.
+   *  **simple_IndividualEnrollment_create.js**
+* Create a simple EnrollmentGroup object.
+   *  **simple_EnrollmentGroup_create.js**
+* Create a couple of IndividualEnrollments and delete them.
+   *  **createdelete_sample.js**
+* Create queries for IndividualEnrollments, EnrollmentGroups and DeviceRegistrationStates that belong to an EnrollmentGroup.
+   *  **query_sample.js**
+* Perform BulkEnrollmentOperations to do various CRUD operations.
+   *  **bulkupdate_sample.js**
+
+
+## How to run the samples
+
+In order to run the device samples you will first need the following prerequisites:
+* Node.js v0.10 or above on your target device. (Check out [Nodejs.org](https://nodejs.org/) for more info)
```

Comment:
>v0.10 [](start = 10, length = 5)

v4. our dependencies are dropping support for older versions and these aren't supported.

Reply:
Fixed.

---
In reply to: [150136312](https://github.com/Azure/azure-iot-sdk-node/pull/142#discussion_r150136312) [](ancestors = 150136312)